tools/utils.py

```diff
@@ -148,28 +149,69 @@ def get_grouped_impfzentren() -> dict:
         result[f"Gruppe {gruppe}"] = impfzentren
     return result
 
+
 def update_available():
-   
     # 2 Zeichen Puffer für zukünftige Versionssprünge
     current_version = get_current_version()
     latest_version = get_latest_version()
 
-
     if latest_version.strip() == current_version.strip():
         return False
     else:
         return True
 
+
 def get_current_version():
     try:
         with open("version.txt") as file:
-                file_contents = file.readlines()
-                current_version = file_contents[0]
-                return current_version
+            file_contents = file.readlines()
+            current_version = file_contents[0]
+            return current_version
     except:
         pass
 
+
 def get_latest_version():
     json_url = 'https://api.github.com/repos/iamnotturner/vaccipy/releases/latest'
     latest_version = requests.get(json_url).json()['tag_name']
     return latest_version
+
+
+def pushover_notification(message: str, kontakt: dict):
```

Comment:
Ich glaube es ist sinnvoll, den `message`-String als letzten Parameter zu übergeben. Das ist üblich, da der oft inline angegeben wird und man sonst die anderen Parameter (in dem Fall `kontakt`) nicht mehr sieht.